### PR TITLE
[INLONG-4040][Manager] Fix hive sink table data path

### LINKED ISSUE #4040: [Bug][Manager] Hive sink table path config error
### What happened

The hive table hdfs path written by sort job does not match with the default database location.
For example the hive table created by manager is /user/hive/warehouse/{db_name}.db/{table_name}.
But the sort job writes to /user/hive/warehouse/{table_name} so query in hive doesn't work.

### What you expected to happen

Correct sort sink path so that data can be correctly retrieved by hive.

### How to reproduce

Configure a hive sink group and run the stream.

### Environment

MacOs

### InLong version

master

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by woofyzhao**
Fixes #4040
Tested in local env.

### CODE REVIEW

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/util/SinkInfoUtils.java [hunk 1]
```diff
@@ -182,13 +183,15 @@ private static HiveSinkInfo createHiveSinkInfo(HiveSinkResponse hiveInfo, List<F
             }).collect(Collectors.toList());
         }
 
-        // dataPath = dataPath + / + tableName
-        StringBuilder dataPathBuilder = new StringBuilder();
+        // dataPath = dataPath + / + dbName + / + tableName
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/util/SinkInfoUtils.java:186]: We use the data path and table name as the real path to writing files.
If changed, maybe occurred an exception in other cases.

Comment (healchow) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/util/SinkInfoUtils.java:186]: Maybe you can change the usage of the dataPath when creating Hive tables.

Comment (woofyzhao) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/util/SinkInfoUtils.java]: Added compatibility for the default db case.

No matter where  you create hive table the table name must be part of the path and must also be specified here to let sort write to the correct file. The default hive approach is fine. All that's left is to take the default database case into account.

**Review by dockerzhang — APPROVED** [review]

**Comment by woofyzhao** [comment]
It turns out that the user should specify full path rather than just warehouse prefix.

**Pull request closed without merging**